app/workers/eject_cargo_worker.rb

```diff
@@ -1,11 +1,7 @@
-class EjectCargoWorker
+class EjectCargoWorker < ApplicationWorker
   # This worker will be run when the user ejects cargo
-
-  include Sidekiq::Worker
-  sidekiq_options retry: false
-
-  def perform(user_id, loader, amount)
-    user = User.find(user_id)
+  def perform(user, loader, amount)
+    user = User.ensure(user)
```

Comment:
Can we rename the var `user` to `user_id `again? Otherwise I will always get a half heart-attack when I see this line :smile:

Reply:
so... this is how that works.. 
- if i pass a User model to `User.ensure()` it returns the model without doing anything
- if i pass an identifier (set by `ensure_by` on the model -- defaults to just `id`) -- it will look it up and return the Model
- if i pass a global_id of the model, it returns the Model

It "ensures" that you have a User.model() no matter what you give it.  
Therefore, the api of method should be the "model" you want to use.

I'm fine with changing it back -- you're preference -- i like the way it reads 
Take a `user` a `loader` and an `amount` and do something

Reply:
Okay